README.md

```diff
@@ -21,19 +21,24 @@ title: Song title
 Song lyrics...
 ```
 
-You can also specify the melody and/or writers:
+You can also specify the melody and/or writers, as well as tags:
 
 ```
 ---
 title: Gurulassa
 melody: Anssi Kela - Puistossa
 writers: Ada
+tags: finnish,medium
 ---
 
 Gurulassa ilta viilenee.
 Topi lämpimikseen koodailee...
 ```
 
+Tags are to be specified as comma-separated values without spaces in between (see above example). If you want to add a tag, append it to `staticData.json`.
```

Comment:
```suggestion
Tags are to be specified as comma-separated values. If you want to add a tag, append it to `staticData.json`.
```

we should support whitespaces, as it's trivial on the parsing side as well as being better contributor experience